### PR TITLE
Return bool as WP_Scripts::do_item does

### PR DESCRIPTION
**Pull request by szepeviktor**
https://developer.wordpress.org/reference/classes/wp_scripts/do_item/

Discovered by @phpstan on Level 4

### CODE REVIEW

File: wp-includes/class-wp-service-worker-scripts.php [hunk 1]
```diff
@@ -148,7 +148,7 @@ public function get_all() {
 	 *
 	 * @param string $handle Handle.
 	 * @param bool   $group Group. Unused.
-	 * @return void
+	 * @return bool
```
Comment (swissspidy) [inline wp-includes/class-wp-service-worker-scripts.php]:
```suggestion
	 * @return bool True on success, false on failure.
```

File: wp-includes/class-wp-service-worker-scripts.php [hunk 2]
```diff
@@ -176,6 +176,8 @@ public function do_item( $handle, $group = false ) {
 			@_doing_it_wrong( 'WP_Service_Worker_Scripts::register', esc_html( $error ), '0.1' ); // phpcs:ignore Generic.PHP.NoSilencedErrors.Discouraged, WordPress.PHP.NoSilencedErrors.Discouraged -- We want the error in the PHP log, but not in the JS output.
 			printf( "console.warn( %s );\n", wp_service_worker_json_encode( $error ) ); // phpcs:ignore WordPress.XSS.EscapeOutput, WordPress.Security.EscapeOutput
 		}
+
+		return $invalid;
```
Comment (swissspidy) [inline wp-includes/class-wp-service-worker-scripts.php]:
```suggestion
		return ! $invalid;
```

The way `WP_Scripts` works, we should return `true` if everything works fine, not the other way around.

Comment (szepeviktor) [reply wp-includes/class-wp-service-worker-scripts.php]: Thank you for noticing!

Done.

**Comment by westonruter** [comment]
The build is failing at no fault of the PR. Once https://github.com/GoogleChromeLabs/pwa-wp/pull/244 is merged, please rebase to re-trigger a build.

**Comment by westonruter** [comment]
I'm not sure why this is still failing. Please pull in the latest changes from `master` to trigger another build.

**Comment by szepeviktor** [comment]
Okay!

**Pull request merged**